bindings/pyroot/cppyy/cppyy-backend/cling/python/cppyy_backend/cmake/FindCppyy.cmake

```diff
@@ -304,13 +304,13 @@ function(cppyy_add_bindings pkg pkg_version author author_email)
 
     # License file
     if("${ARG_LICENSE_FILE}" STREQUAL "")
-        set(ARG_LICENSE_FILE ${CMAKE_SOURCE_DIR}/LICENSE)
+        set(ARG_LICENSE_FILE ${ROOT_MAIN_SOURCE_DIR}/LICENSE)
     endif()
     set(LICENSE_FILE ${ARG_LICENSE_FILE})
 
     # ReadMe file
     if("${ARG_README_FILE}" STREQUAL "")
-        set(ARG_README_FILE ${CMAKE_SOURCE_DIR}/README.rst)
+        set(ARG_README_FILE ${ROOT_MAIN_SOURCE_DIR}/README.rst)
```

Comment:
This path is likely wrong, it should probably be something inside `bindings/pyroot/cppyy/`...